### DISCUSSION TITLE
external link in list

### DISCUSSION THREAD

**Discussion by bilak** [adityatelange/hugo-PaperMod#1246] [Q&A]
Hi,
I'm building some aggregator of blogs and I only need to list them and by clicking on the given post navigate to external url. I've tried to use `canonicalURL` for this purpose but this just displays the url in the post detail. Is there a way how to change the `PermaLink` to some external url?

Thanks

**Comment by RoneoOrg** [comment]
In `baseof.html`:

replace

```
            <a class="entry-link" aria-label="post link to {{ .Title | plainify }}"
              href="{{ .Permalink }}"></a>
```

with

```
	         {{ $href := .RelPermalink }}
            {{ with .Params.redirectionPermalink }}
              {{ with resources.Get . }}
                {{ $href = .RelPermalink }}
              {{ else }}
                {{ $href = . }}
              {{ end }}
            {{ end }}
            <a class="entry-link" aria-label="post link to {{ .Title | plainify }}"
              href="{{ $href }}"></a>
```

then add to your Front Matter the param `redirectionPermalink: https://whatever.com`
